pom.xml

```diff
@@ -67,7 +67,7 @@ THE SOFTWARE.
         <changelist>-SNAPSHOT</changelist>
         <java.level>8</java.level>
         <jenkins.version>2.121.1</jenkins.version>
```

Comment:
May work but 2.138.4 is the minimum recommended version.

Reply:
Okay.